contribs/gmf/src/directives/objecteditingtools.js

```diff
@@ -69,6 +69,7 @@ gmf.module.directive('gmfObjecteditingtools', gmf.objecteditingtoolsDirective);
 
 
 /**
+ * @param {angular.$injector} $injector Main injector.
```

Comment:
Can you add a note about the existence of the configuration object `gmfx.ObjectEditingToolsOptions` in the description of this controller ?